runtime/src/bank/tests.rs

```diff
@@ -2917,10 +2917,10 @@ fn test_bank_tx_compute_unit_fee() {
     );
 }
 
-#[test]
-fn test_bank_blockhash_fee_structure() {
```

Comment:
the original test is invalid. Transaction Fee does not change between banks; It only switch between zero or non-zero based on if cluster is setup for test. Rework the test.